wavefront/server/scripts/test_kms_auth_flow.py

```
@@ -0,0 +1,140 @@
+from flo_cloud.kms import FloKmsService
+from floconsole.constants.auth import AUTH_ROLE_ID
+from floconsole.services.token_service import TokenAlgorithms, TokenService
+
+ISSUER = os.getenv('CONSOLE_JWT_ISSUER', 'https://floconsole.rootflo.ai')
+AUDIENCE = os.getenv('CONSOLE_JWT_AUDIENCE', 'https://floconsole.rootflo.ai')
+PREFIX = os.getenv('CONSOLE_TOKEN_PREFIX', 'fc_')
+
+
+def _dummy_pem_keys() -> tuple[str, str]:
+    with tempfile.NamedTemporaryFile(suffix='.pem', delete=False) as priv:
+        subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)